src/ResourceManager/Automation/Commands.Automation/Common/AutomationClient.cs

```diff
@@ -42,6 +42,15 @@
 using Runbook = Microsoft.Azure.Commands.Automation.Model.Runbook;
 using Schedule = Microsoft.Azure.Commands.Automation.Model.Schedule;
 using Variable = Microsoft.Azure.Commands.Automation.Model.Variable;
+<<<<<<< 5a3f20949198c648832d7ae2bed4ba88898a4a3d
```

Comment:
@robplank Re: conflicts

I still see these in AutomationClient.cs:
`<<<<<<< 5a3f20949198c648832d7ae2bed4ba88898a4a3d`
I think I see it in 2 places?